bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/util/CreateSQLScriptToInsertRegions.kt

```diff
@@ -63,16 +63,4 @@ class CreateSQLScriptToInsertRegions {
 
 }
 
-/**
```

Comment:
Although this whole utility program is not currently used, it was kept for recreating a SQL file if needed in the future.
The main() is part of the program but was commented out because Spring Boot doesn't like two main functions.
So I wouldn't just remove the whole function.

Maybe rename the function to `generate()` and comment it in with a remark that one can add a very simple main() function just calling generate().

Reply:
Done it , putted again with generate and leave the comment about the function on top, will request review again